### PR TITLE
Get every day time arrival method created

### PR DESCRIPTION
**Pull request by Charlie0204**
Created every day time arrival of one week method created with unit test and validators.

### CODE REVIEW

File: test/test-get-one-week-arrival.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+const assert = require('assert');
+
+const request = require('supertest');
+const app = require('../server');
+
+const mongodbMemoryServer = require('./util/mongodb-in-memory');
+const mockData = require('./util/mock-data');
+const mongoClientStub = {};
+const proxyRequire = require('proxyquire');
```
Comment (Miguelsiqueiros) [inline test/test-get-one-week-arrival.js]: We can remove this lines since were no longer stubbing the method that connects the mongodb in memory.

File: src/API/Dashboard/dashboard.js [hunk 2]
```diff
@@ -170,4 +245,20 @@ async function getLastWeeksData(numberOfWeeks){
...
+function groupBy(collection, property){
+    let val, index, values = [], result = [];
+    for (let i = 0; i < collection.length; i++) {
+        val = collection[i][property];
+        index = values.indexOf(val);
+        if (index > -1)
+            result[index].push(collection[i]);
+        else {
+            values.push(val);
+            result.push([collection[i]]);
```
Comment (Miguelsiqueiros) [inline src/API/Dashboard/dashboard.js]: We can simplify this code by grouping the data with the mongo query.

File: test/test-get-one-week-arrival.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+});
+
+const faker = require('faker');
+
+const checkinSchema = require('../src/data/schema/checkin-schema');
+const usersSchema = require('../src/data//schema/user-schema');
+
+const momentjs = require('moment');
+
+describe("Get one week arrival data", () => {
```
Comment (Miguelsiqueiros) [inline test/test-get-one-week-arrival.js]: We should add a test where we have no data and we expect a 200 status code and an empty array.
